about.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+    <link rel="stylesheet" href="about.css" />
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Capstone Project</title>
+  </head>
+  <body>
+    <header>
+      <div class="nav-bar1">
+        
+          <a href="facebook.com/promzzyokechukwu"><i class="fa-brands fa-facebook-f"></i></a>
+          <a href="facebook.com/promzzyokechukwu"><i class="fa-brands fa-twitter"></i></a>
+          <a href="#">Logout</a>
+          <a href="https://promzzykoncepts.github.io/My-Portfolio/">My Page</a>
+          <a href="#">English</a>
+        </div>
+      
+      <nav class="nav-bar">
+
+        
+      <ul class="nav-menu">
+        <li class="nav-item logo">
+          
+          <a href="#"><img class="logo-img" src="images/R.png" alt="logo"/> 
+          <h3>REMOTE GLOBAL  SUMMIT<br>2022</h3></a>
+          
+        </li>
+          <li class="nav-item">
+              <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">Join</a>
+          </li>
+             <li class="nav-item labelT">
+            <a href="" class="nav-link">WFH Remoters..</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+          
+      </div>
+      </nav>
```

Comment:
- [ ] Kindly fix the nav for in the about sectionsince it is very close to the top so make some margin /padding to it 🚀♻ to make your website identical to the original design ✅